### ISSUE TITLE
Schmidt Array pedalboards

### ISSUE THREAD

**Issue by DaveMNL** [PedalPlayground/PedalPlayground.github.io#469]
Hi, even though these are multi-level boards, you could always have a board for, say, an SA 600 base level and an SA600 top level, with the appropriate sizes and cutouts.

https://www.schmidtarray.com/ for more detail and measurements.

**Comment by joseomar18** [comment]
This would be great!

**Comment by Benjamin-RM** [comment]
Yes please, let's get them in there. Perhaps the guys at Schmidt Array would even send the shots over? It could only be positive for them

**Comment by DaveMNL** [comment]
I’ll Martyn to this email thread. Martyn, if you haven’t looked at it yet, PedalPlayground if a great resource for planning and laying out pedalboards. You can work with the team there to add your dimensions and images so people could lay out directly on a SA-675XDM_w, for instance… ;) 

Https://www.pedalplayground.com

Dave

**Comment by Benjamin-RM** [comment]
I think the issue may be that Schmidt Array actually charge clients for planning. So they may see this as eating their lunch.

**Comment by Benjamin-RM** [comment]
Is there anything we can do to help get these on here? I had wondered if we could use SA's own images, as below? 

![image](https://github.com/PedalPlayground/pedalplayground/assets/90400280/544c3bb0-18fd-4409-97f1-15ccb77ad2a0)

**Comment by Benjamin-RM** [comment]
Bumping this request :) Is there anything we (the non-coding / design community) can do to help get these on here? Cheers

**Comment by timswansiger** [comment]
This would be awesome!
